### DISCUSSION TITLE
Welcome to veramo Discussions!

### DISCUSSION THREAD

**Discussion by mirceanis** [decentralized-identity/veramo#290] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community and we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do or hope to do with this framework.

**Comment by trentlarson** [comment]
Hello!  About myself: I am a programmer working for Medici Land Governance, and on the side I enjoy developing systems that could build community.  For my work, I hope to issue and manage DIDs and credentials for both citizen and government actors as they assert and protect land rights; for community projects, I hope to use credentials as claims where individuals can build reputation and even discover connections to people with similar interests and goals (eg. https://github.com/trentlarson/endorser-ch ).

**Reply by jasheal** [reply]
Hi and welcome @trentlarson! 

Thanks for the feedback and participation - 

Sounds great. We look forward to working with you on this as community use cases are very interesting to us too. 

Things will get much more active around here next week as the Veramo team will be back in full swing!
